admin-jobs/app/models/NewsAlertNotification.scala

```diff
@@ -0,0 +1,76 @@
+package models
+
+import java.net.URI
+import java.util.UUID
+
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+import play.api.libs.json.Reads._
+import play.api.data.validation._
+
+import scala.util.{Failure, Success, Try}
+
+case class NewsAlertNotification(id: UUID,
+                                 title: String,
+                                 message: String,
+                                 thumbnailUrl: Option[URI] = None,
+                                 link: URI,
+                                 imageUrl: Option[URI] = None,
+                                 publicationDate : DateTime,
+                                 topics: Set[NewsAlertType]) {
+
+
+  def isOfType(alertType: NewsAlertType) : Boolean = topics.contains(alertType)
+
+}
+
+object NewsAlertNotification {
+
+  // URI serializer/deserializer
+  implicit val uf = new Format[URI] {
+    override def writes(uri: URI): JsValue = JsString(uri.toString)
+    override def reads(json: JsValue): JsResult[URI] = {
+      val error = JsError("Value is expected to convert to URI")
+      json match {
+        case JsString(s) =>
+          Try(URI.create(s)) match {
+            case Success(uri) => JsSuccess(uri)
+            case Failure(_) => error
+          }
+        case _ => error
+      }
+    }
+  }
+
+  // publicationDate (DateTime)
+  implicit val dtf = new Format[DateTime] {
+    private val timeJsonFormatter = ISODateTimeFormat.dateTime().withZoneUTC()
+    override def writes(dt: DateTime): JsValue = JsString(timeJsonFormatter.print(dt))
+    override def reads(json: JsValue): JsResult[DateTime] = {
+      val error = JsError("Value is expected to convert to DateTime")
+      json match {
+        case JsString(s) =>
+          Try(timeJsonFormatter.parseDateTime(s)) match {
+            case Success(dt) => JsSuccess(dt)
+            case Failure(_) => error
+          }
+        case _ => error
+      }
+    }
+  }
+
+  // NewsAlertNotification serializer/deserializer
+  implicit val jWrites = Json.writes[NewsAlertNotification]
+  implicit val jReads: Reads[NewsAlertNotification] = (
+    (__ \ "id").read[UUID] and
+      (__ \ "title").read[String] and
+      (__ \ "message").read[String] and
+      (__ \ "thumbnailUrl").readNullable[URI] and
+      (__ \ "link").read[URI] and
+      (__ \ "imageUrl").readNullable[URI] and
+      (__ \ "publicationDate").read[DateTime] and
+      (__ \ "topics").read[Set[NewsAlertType]].filter(ValidationError("Notification should have at least one topic"))(_.nonEmpty)
```

Comment:
@piuccio 
According to my discussion with Alex it seems like topics for breaking news are actually fixed, or to phrase it differently we (on the frontend) would, at least for now, only care about `breaking/au`, `breaking/us`, `breaking/uk` or `breaking/international` topics.

The objective of this PR is to decouple breaking-news from Fronts by generating the json file used by the website every time a new notification with one of the topic listed above is sent from n10n. At the moment we are not changing how the website deals with Breaking News content.

Let me know if this is not clear.